imagersite/imagerprofile/migrations/0001_initial.py

```diff
@@ -20,6 +21,8 @@ class Migration(migrations.Migration):
                 ('address', models.TextField(max_length=256, blank=True)),
                 ('url', models.URLField(blank=True)),
                 ('photo_type', models.CharField(max_length=256, blank=True)),
+                ('albums', models.ManyToManyField(related_name='albums', null=True, to='photo.Album')),
+                ('photos', models.ManyToManyField(related_name='photos', null=True, to='photo.Photos')),
```

Comment:
You should be able to simply pass the related model class name as the first argument to these constructors without needing the `to` keyword.  Is that not working for you?